client/src/pages/SearchResultsUser.js

```diff
@@ -1,23 +1,53 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
 import { Typography } from "@material-ui/core";
-import SearchList from "../Components/SearchResList";
+import SearchResList from "../Components/SearchResList";
 import { useLocation } from "react-router-dom";
 
 function SearchResultsUser() {
 
-// Will get the query from the URL: https://codesandbox.io/s/react-router-query-parameters-mfh8p?from-embed
-  const useQuery = () => {
+const [results, setResults] = useState(null);  
+const [error, setError] = useState(null);
+const useQuery = () => {
     return new URLSearchParams(useLocation().search);
   }
 
-  const query = useQuery().get("query");
+//"https://codesandbox.io/s/react-router-query-parameters-mfh8p?from-embed"
+  useEffect(()=>{
+      fetch(events)
```

Comment:
I would like to see a check here before either fetch is called that is something like 
```switch(query) {
     case "events":
           fetch(events)...
          break;
     case "organizations":
            fetch(organizations)...
            break;
     default:
           return; // Should do nothing because the query didn't match either expected option
  }```